layout.deskintl.h

```diff
@@ -250,6 +275,101 @@ static struct key keys_special[] = {
   {"", "", 0.0, Last},
 };
 
+static struct key keys_cyrillic[] = {
+  {"Esc", "Esc", 1.25, Code, KEY_ESC, .scheme = 1},
+  {"F1", "F1", 1.0, Code, KEY_F1, .scheme = 1},
+  {"F2", "F2", 1.0, Code, KEY_F2, .scheme = 1},
+  {"F3", "F3", 1.0, Code, KEY_F3, .scheme = 1},
+  {"F4", "F4", 1.0, Code, KEY_F4, .scheme = 1},
+  {"F5", "F5", 1.0, Code, KEY_F5, .scheme = 1},
+  {"F6", "F6", 1.0, Code, KEY_F6, .scheme = 1},
+  {"F7", "F7", 1.0, Code, KEY_F7, .scheme = 1},
+  {"F8", "F8", 1.0, Code, KEY_F8, .scheme = 1},
+  {"F9", "F9", 1.0, Code, KEY_F9, .scheme = 1},
+  {"F10", "F10", 1.0, Code, KEY_F10, .scheme = 1},
+  {"F11", "F11", 1.0, Code, KEY_F11, .scheme = 1},
+  {"F12", "F12", 1.0, Code, KEY_F12, .scheme = 1},
+  {"Del", "Del", 1.25, Code, KEY_DELETE, .scheme = 1},
+  {"", "", 0.0, EndRow},
+
+
```

Comment:
A small adjustment: Replace the two blank lines with one blank line here.

Reply:
thanks! applied